operations/mimir-tests/test-single-to-multi-zone-distributor-migration.jsonnet

```diff
@@ -0,0 +1,13 @@
+// Based on test-multi-zone.jsonnet.
+(import 'test-multi-zone.jsonnet') {
```

Comment:
Base it on the multi-zone distributor test so that it's trivial to compare, thanks.